assets/js/theme/common/product-details.js

```diff
@@ -393,6 +393,13 @@ export default class ProductDetails {
             viewModel.$addToCart.prop('disabled', false);
             viewModel.$increments.prop('disabled', false);
         }
+
+        // If Bulk Pricing rendered HTML is available
+        if (content) {
```

Comment:
does this work?  i thought it was grabbing a field off of content... like this instead:

```
updateView(data, content) { // content will be {} if response.content is empty above
```
...
```
if (content.bulkPricingContent) { // or whatever it's called
  viewModel.$bulkPricing.html(content.bulkPricingContent)
```

Reply:
Content returns a blank string which evaluates to no if there are no tiers.